spring-cloud-generator/WORKSPACE

```diff
@@ -0,0 +1,58 @@
+workspace(name = "spring_cloud_generator")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# Maven install
+
+RULES_JVM_EXTERNAL_TAG = "4.5"
+RULES_JVM_EXTERNAL_SHA = "b17d7388feb9bfa7f2fa09031b32707df529f26c91ab9e5d909eb1676badd9a6"
+
+http_archive(
+    name = "rules_jvm_external",
+    strip_prefix = "rules_jvm_external-%s" % RULES_JVM_EXTERNAL_TAG,
+    sha256 = RULES_JVM_EXTERNAL_SHA,
+    url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % RULES_JVM_EXTERNAL_TAG,
+)
+
+load("@rules_jvm_external//:repositories.bzl", "rules_jvm_external_deps")
+
+rules_jvm_external_deps()
+
+load("@rules_jvm_external//:setup.bzl", "rules_jvm_external_setup")
+
+rules_jvm_external_setup()
+
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+
+
+# Spring generator
+_spring_cloud_generator_version = "4.1.4-SNAPSHOT" # {x-version-update:spring-cloud-gcp:current}
```

Comment:
I think this version needs to be in sync with the latest spring-cloud-gcp version? I'm actually not sure how it worked with the wrong version.

Reply:
You're right - thanks for catching this! I'm also confused to how this slipped past and kept working without the version being in sync - will take a look.

Reply:
Ah, this may have worked with the incorrect version because the setup here is redundant/unused in the generation process. I've trimmed away most of this file in [87df937](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1686/commits/87df9371a34c499de8e5df0f799c8263440462ba), with a quick explanation below:

When initially following the `gapic-generator-java` approach, I was under the impression that the generator WORKSPACE needed a `maven_install()` for the end-to-end generation, and followed most of this setup. 

On a second look at `gapic-generator-java/WORKSPACE`, I realize that it's [fetching a snapshot jar](https://github.com/googleapis/gapic-generator-java/blob/4054bc668daf797a8fce798843e0ec4cda669faa/WORKSPACE#L63), so probably just to allow for bazel invocations from within the repo (for integration tests?), while `googleapis/WORKSPACE` [fetches a published version](https://github.com/googleapis/googleapis/blob/bab324b44955494aebc6d572cee061202c4f811f/WORKSPACE#L268). In the spring codegen process, because we are not publishing the generator, the local copy of `googleapis/WORKSPACE` is already being modified with a [maven_install()](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/generator-migration-test/spring-cloud-generator/scripts/resources/googleapis_modification_string.txt) block to fetch the [locally built](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/87df9371a34c499de8e5df0f799c8263440462ba/spring-cloud-generator/scripts/generate-all.sh#L47-L48) generator as part of scripts. The version was correct there, which is why the generation workflow had worked.

This setup may be needed again when adding integration tests for the spring generator, but I will remove it for now given the additional version management.